diesel/src/migrations/migration.rs

```diff
@@ -97,6 +98,31 @@ pub fn version_from_path(path: &Path) -> Result<String, MigrationError> {
         })
 }
 
+#[allow(missing_debug_implementations)]
```

Comment:
Surprised this didn't trigger a missing copy implementations as well. Should this type implement `Copy`?

Reply:
Tbh I'm no1 100% sure if it _should_ as I'm unfamiliar with the case in which it's necessary. If you think it should I can add it as well.

Reply:
This type should implement `Copy`